modules/core/04-channel/types/codec.go

```diff
@@ -50,3 +52,35 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 // The actual codec used for serialization should be provided to x/ibc/core/04-channel and
 // defined at the application level.
 var SubModuleCdc = codec.NewProtoCodec(codectypes.NewInterfaceRegistry())
+
+// UnpackAcknowledgement unpacks an Any into an Acknowledgement. It returns an error if the
+// acknowledgement can't be unpacked into a Acknowledgement.
+func UnpackAcknowledgement(any *codectypes.Any) (exported.Acknowledgement, error) {
```

Comment:
@colin-axner I will make a follow-up PR with these changes into main if we decide to go this route.